src/tests/IntegrationTests/Command/CleanerCommandTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace SymfonyCasts\Bundle\tests\IntegrationTests\Command;
+
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ * @author Ryan Weaver   <ryan@symfonycasts.com>
+ */
+class CleanerCommandTest extends KernelTestCase
+{
+    public function testGarbageCollection(): void
+    {
+        //@TODO Remove skip after setting up integration tests
+        $this->markTestIncomplete('IntegrationTests not setup yet.');
```

Comment:
Do you still want to add an integration test?

Reply:
yes but ready to do so now.. we need either functional or integration tests (possibly both) to ensure our configuration is working as intended.. can discuss that later..

removed the test for now, will implement it in the future..

Reply:
Awesome - keep it on the list - I also like to have at least one integration test that shows that the container will boot up correctly with the bundle installed & that any service with autowireable aliases can be fetched & instantiated via autowiring - like this here https://github.com/symfony/webpack-encore-bundle/blob/5c0f659eceae87271cce54bbdfb05ed8ec9007bd/tests/IntegrationTest.php#L143-L149